knowledge-base/capturing-traffic-from-terminal.md

```diff
@@ -15,33 +14,63 @@ previous_url: fiddler-macos-terminal
 | Product   | Fiddler Everywhere |
 | Product Version | 1.0.0 and above  |
 | Operating System | macOS, Windows, Linux  |
-| 3rd-party tools | macOS Terminal, iTerm2, Command Prompt, Bash, Fish, Zsh |
+| 3rd-party tools | macOS Terminal, iTerm2, Command Prompt, PowerShell, Bash, Fish, Zsh |
 
 ## Description
 
-How can I set the Fiddler Everywhere proxy for commonly used command-line tools like the default [macOS Terminal](https://en.wikipedia.org/wiki/Terminal_(macOS)), [iTerm2](https://www.iterm2.com/), Command Prompt (Windows), [Bash](https://www.gnu.org/software/bash/) and other terminals or shell applications?
+How can I set the Fiddler Everywhere proxy for commonly used command-line tools like [Windows PowerShell](https://learn.microsoft.com/en-us/powershell/scripting/overview?view=powershell-7.3), [macOS Terminal](https://en.wikipedia.org/wiki/Terminal_(macOS)), [iTerm2](https://www.iterm2.com/), Command Prompt (Windows), [Bash](https://www.gnu.org/software/bash/) and other terminals or shell applications?
 
 
-### Setting the Fiddler Proxy Explicitly
+## Capturing Traffic from Preconfigured Terminal
 
-Terminals and shell applications like the [Terminal](https://en.wikipedia.org/wiki/Terminal_(macOS)), [iTerm2](https://www.iterm2.com/), Command Prompt (Windows) are capable of executing HTTP and HTTPS requests through third-party tools like CURL. However, in the typical case, these tools won't use the system proxy.
+Fiddler Everywhere can automatically start a preconfigured terminal instance through the **>_ Terminal** button in the **Live Traffic** toolbar.
 
-For Fiddler Everywhere to capture traffic that passes through them, they need to explicitly set the Fiddler Everywhere proxy. You can use the `export` command (for macOS and Linux) or the `set` command (for Windows) to achieve that. The `export` command will generate an environmental variable that will be included in a child process environment. It does not affect other already-existing settings.
+![Use the Terminal button to start preconfigured terminal instane](../images/get-started/get-started-terminal.png)
+
+You can set your preferred terminal application through **Settings > Terminal > Default Terminal**. The following options are available:
+
+- **Command Prompt**&mdash;Available only on Windows.
+- **Windows PowerShell**&mdash;Available only on Windows.
+- **PowerShell**&mdash;Available only on Windows.
+- **Terminal application**&mdash;Available on macOS and Linux. This option will use the default shell environment, for example, **bash**, **zsh**, or **sh**.
+
+The Fiddler's preconfigured terminal instance automatically proxies all requests made by **curl** or **NodeJS** libraries (like **https**, **request**, **axios**, etc.).
```

Comment:
I'd add "and all applications, which respect the HTTP_PROXY AND HTTPS_PROXY environment variables. Node and Curl are configured to trust the Fiddler Certificate."